lib/rbui/checkbox/checkbox_group_controller.js

```diff
@@ -0,0 +1,23 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+  static targets = ['checkbox'];
+
+  connect() {
+    if (this.element.hasAttribute("data-required")) {
+      this.checkboxTargets.forEach(checkbox => {
+        checkbox.required = true;
+      });
+    }
+  }
+
+  onChange(event) {
+    if (this.element.hasAttribute("data-required")) {
```

Comment:
can we use just the `required` attribute?

Reply:
no =/